Sources/protoc-gen-swiftgrpc/Generator-Client.swift

```diff
@@ -216,6 +227,44 @@ extension Generator {
           outdent()
           println("}")
         }
+        if rxSwiftCode {
+          println("/// RxSwift. Unary.")
+          println("\(access) func \(methodFunctionName)(_ request: \(methodInputName)) -> Observable<\(methodOutputName)> {")
+          indent()
+          println("return \(methodFunctionName)(request, metadata: metadata)")
+          outdent()
+          println("}")
+          println("/// RxSwift with metadata. Unary.")
+          println("\(access) func \(methodFunctionName)(_ request: \(methodInputName), metadata: Metadata) -> Observable<\(methodOutputName)> {")
+          indent()
+          println("return Observable.just(request)")
+          indent()
+          println(".flatMap { request -> Observable<\(methodOutputName)> in")
+          indent()
+          println("return Observable.create { observer in")
+          indent()
+          println("do {")
+          indent()
+          println("let response = try \(callName)Base(self.channel)")
+          indent()
+          println(".run(request: request, metadata: metadata)")
```

Comment:
I think in this case it would be better to use the non-blocking variant of the unary call, and in the corresponding completion handler call `response.onNext/onError` as needed. Currently this might actually block the calling thread.